qutebrowser/config/websettings.py

```diff
@@ -198,6 +198,15 @@ def init(args: argparse.Namespace) -> None:
                                 pattern=urlmatch.UrlPattern(pattern))
 
 
+def clear_private_data() -> None:
+    """Clear cookies, cache and related data for private browsing sessions."""
+    if objects.backend == usertypes.Backend.QtWebEngine:
+        from qutebrowser.browser.webengine import webenginesettings
+        webenginesettings.init_private_profile()
+    else:
```

Comment:
Agreed, although I just copied the pattern used in another two functions in that file: check for QtWebEngine, if not assume QtWebKit. Since I'm here I made the same adjustment to those functions as well.

Thanks for your comments!